general/networking/traefik2.md

```diff
@@ -0,0 +1,267 @@
+---
+uid: network-reverse-proxy-traefik2
+title: Traefik2
```

Comment:
If this title is user visible I would change it to `Traefik v2` or similar

Reply:
The short title is not user visible.  The title below it is, similar to as you suggest `Traefik v2.x`, and is visible to the end user.  This title is more to match the table of contents and the doc filename with a short title.

Reply:
Actually in hindsight I agree.  Changing.